versioned_plugins.rb

```diff
@@ -429,6 +432,41 @@ def load_alias_definitions_for_target_plugins(plugin_names_by_type)
 
     aliases
   end
+
+  def fetch_stack_versions
+    stack_versions = Net::HTTP.get(URI.parse(VERSIONS_URL))
+    puts "Logstash version: #{get_logstash_version(stack_versions)}\n"
+    puts "ECS version: #{get_ecs_version(stack_versions)}\n"
+  end
+
+  def get_logstash_version(stack_versions)
+    version = stack_versions[/\:logstash_version: (.*?)\n/, 1]
+    @logstash_version = version.strip unless version.nil?
+  end
+
+  def get_ecs_version(stack_versions)
+    version = stack_versions[/\:ecs_version: (.*?)\n/, 1]
+    @ecs_version = version.strip unless version.nil?
+  end
```

Comment:
Minot nitpick: since the methods are named get_* I'd suggest moving the setting of the instance variables to the caller:

```suggestion
  def fetch_stack_versions
    stack_versions = Net::HTTP.get(URI.parse(VERSIONS_URL))
    @logstash_version = get_logstash_version(stack_versions)
    puts "Logstash version: #{@logstash_version}\n"
    @ecs_version = get_ecs_version(stack_versions)
    puts "ECS version: #{@ecs_version}\n"
  end

  def get_logstash_version(stack_versions)
    version = stack_versions[/\:logstash_version: (.*?)\n/, 1]
    version.strip unless version.nil?
  end

  def get_ecs_version(stack_versions)
    version = stack_versions[/\:ecs_version: (.*?)\n/, 1]
    version.strip unless version.nil?
  end
```